docs/en/deploy/Running-Alluxio-Using-Vagrant.md

```diff
@@ -5,41 +5,127 @@ nickname: Alluxio on Vagrant
 group: Deploying Alluxio
 priority: 5
 ---
-TODO(david): Outline
-  - Table of Contents
-  - Introduction
-  - Prerequisite for using Vagrant
-  - Deploy using Vagrant on Virtual Box
-    - Launch Cluster
-    - Access / Verify the Cluster
-    - Destroy 
-  - Deploy using Vagrant on EC2
-    - Launch Cluster
-    - Access / Verify the Cluster
-    - Destroy
-    - Advanced Tips
-      -Spot Instance
-  - Deploy using Vagrant on GCE
-    - Launch Cluster
-    - Access / Verify the Cluster
-    - Destroy
-
-TODO(david): combine content from Deploy on EC2, GCE, Virtual Box using Vagrant 
-
 * Table of Contents
 {:toc}
 
-Alluxio can be deployed on Amazon EC2 using the [Vagrant scripts](https://github.com/alluxio/alluxio/tree/master/deploy/vagrant) that come with Alluxio. The scripts let you create, configure, and destroy clusters that come automatically configured with [Amazon S3](https://s3.amazonaws.com/).
+## Introduction
```

Comment:
not sure how applicable, but should we keep the outline of this doc consistent with the deploy Alluxio template ?

Reply:
it is difficult to keep it consistent with the deploy Alluxio template because they are very different, and we are  describing deploying procedures for three different scenarios.

Reply:
ack